.gitlab/test_cpp.sh

```diff
@@ -74,9 +63,11 @@ cd ${INSTALL_DIR}
 ./bin/test_plugin
 
 # Run NIXL client-server test
-./bin/nixl_test target 127.0.0.1 1234&
+nixl_test_port=$(get_next_server_port)
+
+./bin/nixl_test target 127.0.0.1 $nixl_test_port&
```

Comment:
from quick check there are places in gtest that also open TCP server socket:
1. search for getRandomPort
2. search for getPort 

1 and 2 eventually affect nixlMetadataStream::setupStream